apps/nuxt3-ssr/components/AppHeader.vue

```diff
@@ -4,14 +4,14 @@ if (useRoute) {
   schema = useRoute().params.schema
 }
 const menu = [
-  { label: "Home", link: "#" },
-  { label: "Variables", link: "#", highlight: true },
-  { label: "Cohorts", link: "#" },
-  { label: "Networks", link: "#" },
-  { label: "Statistical Methods", link: "#" },
-  { label: "Tables", link: "#" },
-  { label: "Manuals", link: "#" },
-  { label: "About", link: "#" },
+  // { label: "Home", link: "#" },
```

Comment:
the umcg mvp does not use these things, but i would like to keep them in de code to avoid having to search the git history when we do want to use them. The umcg mvp impl only uses a small part of the pixelpillow design. In general i like to just delete and use the git history but in this case i propose to comment out part of the design that is not being used